colour/models/dataset/aces.py

```diff
@@ -0,0 +1,436 @@
+"""
+
+
+def _aces_cc_transfer_function(value):
+    """
+    Defines the *ACEScc* colourspace transfer function.
+
+    Parameters
+    ----------
+    value : numeric
+        Value.
+
+    Returns
+    -------
+    numeric
+        Companded value.
+    """
+
+    if value < 0:
+        return (np.log2(2 ** -15 * 0.5) + 9.72) / 17.52
+    elif value < 2 ** -15:
+        return (np.log2(2 ** -16 + value * 0.5) + 9.72) / 17.52
+    else:
+        return (np.log2(value) + 9.72) / 17.52
+
+
+def _aces_cc_inverse_transfer_function(value):
+    """
+    Defines the *ACEScc* colourspace inverse transfer function.
+
+    Parameters
+    ----------
+    value : numeric
+        Value.
+
+    Returns
+    -------
+    numeric
+        Companded value.
+    """
+
+    if value < (9.72 - 15) / 17.52:
+        return (2 ** (value * 17.52 - 9.72) - 2 ** -16) * 2
+    elif (9.72 - 15) / 17.52 < value < (np.log2(65504) + 9.72) / 17.52:
+        return 2 ** (value * 17.52 - 9.72)
+    else:
+        return 65504
+
+
+ACES_CC_TRANSFER_FUNCTION = _aces_cc_transfer_function
+"""
+Transfer function from linear to *ACEScc* colourspace.
+
+ACES_CC_TRANSFER_FUNCTION : object
+"""
+
+ACES_CC_INVERSE_TRANSFER_FUNCTION = (
+    _aces_cc_inverse_transfer_function)
+"""
+Inverse transfer function from *ACEScc* colourspace to linear.
+
+ACES_CC_INVERSE_TRANSFER_FUNCTION : object
+"""
+
+ACES_CC_COLOURSPACE = RGB_Colourspace(
+    'ACEScc',
+    AP1,
+    ACES_WHITEPOINT,
+    ACES_ILLUMINANT,
+    AP1_TO_XYZ_MATRIX,
+    XYZ_TO_AP1_MATRIX,
+    ACES_CC_TRANSFER_FUNCTION,
+    ACES_CC_INVERSE_TRANSFER_FUNCTION)
+"""
+*ACEScc* colourspace, a working space for color correctors, target for ASC-CDL
+values created on-set.
+
+ACES_CC_COLOURSPACE : RGB_Colourspace
+"""
+
+ACES_PROXY_10_CONSTANTS = Structure(
+    CV_min=64,
+    CV_max=940,
+    steps_per_stop=50,
+    mid_CV_offset=425,
+    mid_log_offset=2.5)
+"""
+*ACESproxy* 10 bit colourspace constants.
+
+ACES_PROXY_10_CONSTANTS : Structure
+"""
+
+ACES_PROXY_12_CONSTANTS = Structure(
+    CV_min=256,
+    CV_max=3760,
+    steps_per_stop=200,
+    mid_CV_offset=1700,
+    mid_log_offset=2.5)
+"""
+*ACESproxy* 12 bit colourspace constants.
+
+ACES_PROXY_12_CONSTANTS : Structure
+"""
+
+ACES_PROXY_CONSTANTS = CaseInsensitiveMapping(
+    {'10 Bit': ACES_PROXY_10_CONSTANTS,
+     '12 Bit': ACES_PROXY_12_CONSTANTS})
+"""
+Aggregated *ACESproxy* colourspace constants.
+
+ACES_PROXY_CONSTANTS : CaseInsensitiveMapping
+    {'10 Bit', '12 Bit'}
+"""
+
+
+def _aces_proxy_transfer_function(value, bit_depth='10 Bit'):
+    """
+    Defines the *ACESproxy* colourspace transfer function.
+
+    Parameters
+    ----------
+    value : numeric
+        Value.
+    bit_depth : unicode, optional
+        {'10 Bit', '12 Bit'}
+        *ACESproxy* bit depth.
+
+    Returns
+    -------
+    numeric
+        Companded value.
+    """
+
+    constants = ACES_PROXY_CONSTANTS.get(bit_depth)
+
+    float_2_cv = lambda x: max(constants.CV_min,
```

Comment:
Yes most likely! I decided to leave it as is because it is implemented this way in the Technical Bulletin.